coopAndLastFM2.js

```diff
@@ -135,7 +140,9 @@ var app = {
 };
 
 // global variable to hold onto the image
-var theImage;
+
+//add variables for songTitle string and font string
+var theImage, font, songTitle;
 
```

Comment:
global variables for the image, font, songtitle (and eventually font size? font color? whatever you want to change!)